Forms/Main.cs

```diff
@@ -775,7 +776,7 @@ private void ShowDownloadInfo(int epid)
 
                     buttons.Add(this.ButtonDelete);
                     infoText += Environment.NewLine + "Play count: " + info.PlayCount.ToString(CultureInfo.CurrentCulture);
-
+                    infoText += Environment.NewLine + "File path: " + info.DownloadPath.ToString(CultureInfo.CurrentCulture);
```

Comment:
I think this may have crept into this PR by mistake...

Reply:
Sort of, was having some issue with file paths for some podcasts, so added that to easily view. Will remove form this PR. Thoughts on whether it is useful to others (eg worth adding as future enhancement?)

Reply:
In *theory*, the file path should be something that the user doesn't have to worry about too much, so I think we should leave this to one side for the moment.